walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,92 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
 
+/** Find available meeting time slots */
 public final class FindMeetingQuery {
+  /** Get events of requested attendees */
+  private List<Event> getRelevantEvents(Collection<Event> events, MeetingRequest request){
+    List<Event> relevantEvents = new ArrayList<>();
+    Iterator<Event> iter = events.iterator();
+    Event currEvent;
+    Collection<String> requestedAttendee = request.getAttendees();
+    Set<String> meetingAttendee = new HashSet<>();
```

Comment:
Since the set is different from one event to another, this and the meetingAttendee.clear() line can be replaced by a Set<String> attendee in the while/for loop. If so, it's re-instantiated each time. The performance regression is trivial so it's fine.

Reference: https://docs.oracle.com/javase/specs/jls/se7/html/jls-6.html#jls-6.4